feature/gnoi/tests/complete_chassis_reboot/complete_chassis_reboot_test.md

```diff
@@ -0,0 +1,41 @@
+# gNOI-3.1: Complete Chassis Reboot
+
+## Summary
+
+Validate gNOI RPC can reboot entire chassis
+
+## Procedure
+
+
+* Configure ATE port-1 connected to DUT port-1 with the relevant IPv4 and IPv6 addresses.
```

Comment:
This is from the test code comment. This markdown should adhere to the procedure from the test plan.

Looks like the test code only somewhat follows the test plan. At least I could identify the no delay and N second delay cases, but for example the uptime and ports are not checked. Only boot-time is checked, but boot time is not uptime.

These discrepancies need to be recoded in this markdown like this:

* Issue `gnoi.system` `Reboot` RPC...
  * Validate that system uptime ...
    * TODO: test code currently checks boot-time instead of uptime.
  * TODO: Validate that all connected ports are disabled and re-enabled.

Reply:
Done!